smartgymapi/lib/validation/sport_scheme.py

```diff
@@ -1,8 +1,26 @@
-from marshmallow import Schema, fields
+from dateutil.parser import parse
+
+from marshmallow import Schema, fields, ValidationError, validates
 
 
 class SportScheduleSchema(Schema):
     id = fields.Str(dump_only=True)
-    user_id = fields.UUID(required='User id is required')
+    user_id = fields.UUID(required='User id is required', default=None)
```

Comment:
dus hij is required maar toch ook weer niet door een default? user_id hoort sowieso uit de sessie te komen.